README.md

```diff
@@ -64,6 +64,7 @@ configuration file.
 |                                 **_Cobalt2_**<br>[source](https://github.com/wesbos/cobalt2/tree/master/Cobalt2)                                  |               ![Cobalt2](images/Cobalt2.png)               |
 |                                 **_cyber_punk_neon_**<br>[source](https://github.com/Roboron3042/Cyberpunk-Neon)                                  |       ![cyber_punk_neon](images/cyber_punk_neon.png)       |
 |                                                **_darcula_**<br>[source](https://draculatheme.com)                                                |               ![darcula](images/darcula.png)               |
+|                                                **_darcula_plus_**<br>[source](https://draculatheme.com)                                                |               ![darcula_plus](images/darcula_plus.png)               |
```

Comment:
It's dracula, not darcula, right?

Also would you mind deleting the other theme that made the same spelling mistake? We have a dracula theme in the repo and that doesn't seem much different.

Reply:
Made the suggested changes. Do let me know if I need to make any more changes.